lib/linux_admin/security/audit_rules.rb

```diff
@@ -0,0 +1,112 @@
+module LinuxAdmin
+  class Security
+    class AuditRules
+      include LinuxAdmin::Common
+      include Security::Common
+      CONF_FILE = "/etc/audit/rules.d/audit.rules"
+
+      SCAP_FILESYSTEM_RULES = [
+        ["/etc/localtime",         "wa", "audit_time_rules"],
+        ["/etc/group",             "wa", "audit_account_changes"],
+        ["/etc/passwd",            "wa", "audit_account_changes"],
+        ["/etc/gshadow",           "wa", "audit_account_changes"],
+        ["/etc/shadow",            "wa", "audit_account_changes"],
+        ["/etc/security/opasswd",  "wa", "audit_account_changes"],
+        ["/etc/selinux/",          "wa", "MAC-policy"],
+        ["/etc/sudoers",           "wa", "actions"],
+        ["/sbin/insmod",           "x",  "modules"],
+        ["/sbin/rmmod",            "x",  "modules"],
+        ["/sbin/modprobe",         "x",  "modules"],
+        ["/etc/issue",             "wa", "audit_network_modifications"],
+        ["/etc/issue.net",         "wa", "audit_network_modifications"],
+        ["/etc/hosts",             "wa", "audit_network_modifications"],
+        ["/etc/sysconfig/network", "wa", "audit_network_modifications"]
+      ]
+
+      SCAP_SYSTEM_CALL_RULES = [
+        [
+          "always",
+          "exit",
+          %w(sethostname setdomainname),
```

Comment:
Sorry, this seems strange to me.  I'd prefer to see the ```[ ]``` syntax instead of the %w here.